python/README.md

```diff
@@ -36,22 +36,47 @@ DEBUG=true MOMENTO_AUTH_TOKEN=<YOUR_TOKEN> pipenv run python example.py
 DEBUG=true MOMENTO_AUTH_TOKEN=<YOUR_TOKEN> pipenv run python example_async.py
 ```
 
+## Running the Example Using requirements.txt
```

Comment:
This is an awesome touch, thanks for including it.  I might title this section `... Using pip` though, and then mention requirements.txt only in the content rather than in the header.  just in case some users are familiar with `pip` but not familiar with `requirements.txt`.

Reply:
Good suggestion-- I've changed it.